installData.json

```diff
@@ -262,9 +262,10 @@
 					],
 					"fileOverrides" : [
 						{"name": "movie",   "os": ["mac", "linux"], "url":"https://07th-mod.com/rikachama/video/Meakashi-Movie_UNIX.7z"},
-						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p3", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_steam_win.7z"},
-						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p1", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_gog_win.7z"},
-						{"name": "ui",      "os": ["mac", "linux"], "unity": "5.5.3p3", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_steam_unix.7z"}
+						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p3", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=win&unity=5.5.3p3"},
+						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p1", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=win&unity=5.5.3p1"},
+						{"name": "ui",      "os": ["mac", "linux"], "unity": "5.5.3p3", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=unix&unity=5.5.3p3"}
```

Comment:
The build failed cause you're missing a comma here (and possibly other things as well, didn't look through it in detail)